api/src/member/views.py

```diff
@@ -21,6 +22,12 @@ def send_access_token(redirect=Arg(str, required=False), user_identification: st
     )
 
 
+@service.route("/send_updated_member_info", method=POST, permission=PUBLIC)
+def send_updated_member_info(member_number: str = Arg(str)):
+    """Send email to user with information that personal information has been updated."""
+    return send_updated_member_info_email(member_number)
```

Comment:
The member number should be an integer, shouldn't it?

Reply:
Also. This should be the `member_id` not `member_number`. They are different.

Reply:
The member_number is used in the function ` get_member_by_user_identification` which takes a string, user_identification, which is either a member_number (not member_id) or an email. I missed pushing the script calling that function though... I will add that!